ios/TrueSheetView.swift

```diff
@@ -19,7 +19,7 @@ class TrueSheetView: UIView, RCTInvalidating, TrueSheetViewControllerDelegate {
 
   private var isPresented = false
   private var activeIndex: Int?
-  private var bridge: RCTBridge
+  private var bridge?: RCTBridge
```

Comment:
I'm getting error in xCode on this line.

This works
```
private var bridge: RCTBridge?
```

Reply:
Weird, not sure how this happened. Fixed